### DISCUSSION TITLE
Advanced Options

### DISCUSSION THREAD

**Discussion by DmiRials** [rustdesk/rustdesk-server-pro#212] [Q&A] [answered]
I saw that you can specify additional options.
Can I find out in more detail how it works or what it can be used for?
Is it possible to specify a token for a user there to make a client for a specific user?

**Comment by rustdesk** [comment]
Will be ready in doc

**Reply by ConGp20** [reply]
is there something new to this topic?

**Reply by nicedevil007** [reply]
also interested in this, I hope that we can adjust settings like "allow filetransfer" or "show remote cursor"...

**Reply by devNull2and1** [reply]
@rustdesk 

waiting for default / overwrite settings too...

```
theme = 'dark'
enable-check-update = 'N'
show_monitors_toolbar = 'Y'
show_remote_cursor = 'Y'
zoom-cursor = 'Y'
enable_file_transfer = 'Y'
view_style = 'adaptive'
collapse_toolbar = 'Y'
direct-server = 'Y'
allow-remote-config-modification = 'Y'
enable-audio = 'N'
```

Also Missing Custom Server Settings for ID Server, there is only Host, Key and Api. It should be like this:

Host: `myrelayserver.domain.com` Port: `443`
ID: `myidserver.domain.com` Port: `443`
Api: `myapiserver.domain.com` Port: `443`

Behind this domains diffrent public IP Adresse:
`myrelayserver.domain.com` -> `1.2.3.4`
`myidserver.domain.com` -> `1.2.3.5`
`myapiserver.domain.com` -> `1.2.3.6`

This way rustdesk can work on non blocked ports. (21xxx) blocked on enterprise networks.

all above should be possible to brand into customized exe. 
all above should be should be possible to configure with Console -> Strategies too.

not all settings are configurable with Console :(

**Reply by rustdesk** [reply]
Give you some clue about this,

For display option, please prepend `$$`.

https://github.com/rustdesk/rustdesk/blob/6ba6a16836e7c78e574d272b6f03707c455c894f/src/common.rs#L1544

![image](https://github.com/rustdesk/rustdesk-server-pro/assets/71636191/968e1c56-a9c3-4be0-8c9d-0e72c194f563)


For local option, please prepend `$`.

https://github.com/rustdesk/rustdesk/blob/6ba6a16836e7c78e574d272b6f03707c455c894f/src/common.rs#L1549

![image](https://github.com/rustdesk/rustdesk-server-pro/assets/71636191/68e31e6b-5fbb-4f4f-92b2-9f39d463cfa3)

**Reply by devNull2and1** [reply]
Just tried my shotgun:

```
show_monitors_toolbar=Y
show_remote_cursor=Y
zoom-cursor=Y
enable_file_transfer=Y
view_style=adaptive
collapse_toolbar=Y
enable-check-update=N
temporary-password-length=10
direct-server=Y
allow-remote-config-modification=Y
allow-remove-wallpaper=Y
enable-audio=N
enable-open-new-connections-in-tabs=N
theme=dark
$show_monitors_toolbar=Y
$show_remote_cursor=Y
$zoom-cursor=Y
$enable_file_transfer=Y
$view_style=adaptive
$collapse_toolbar=Y
$enable-check-update=N
$temporary-password-length=10
$direct-server=Y
$allow-remote-config-modification=Y
$allow-remove-wallpaper=Y
$enable-audio=N
$enable-open-new-connections-in-tabs=N
$theme=dark
$$show_monitors_toolbar=Y
$$show_remote_cursor=Y
$$zoom-cursor=Y
$$enable_file_transfer=Y
$$view_style=adaptive
$$collapse_toolbar=Y
$$enable-check-update=N
$$temporary-password-length=10
$$direct-server=Y
$$allow-remote-config-modification=Y
$$allow-remove-wallpaper=Y
$$enable-audio=N
$$enable-open-new-connections-in-tabs=N
$$theme=dark
```

ooops exe dont start with this anymore. i think i wait till documentation is there.
Security Settings should be branded too. 

Audio is enabled in standard, so, mostly if you do quick support, You hear your customer in your webmeeting/phone and over rustdesk. the power of echos are incoming then :)

**Reply by rustdesk** [reply]
A lot of tests need to be done. So this is not just a doc issue.

**Reply by devNull2and1** [reply]
uhhhhh, outch, i understand. for me it is a bit weird which settings belong to display, general and how it is done for security settings. maybe i should not paste it in override, maybe i start with small tests and a few vars.

**Comment by rustdesk** [comment, marked as answer]
We will start to work on this after we finish Linux custom build.

**Reply by rustdesk** [reply]
Since we released linux yesterday, we may start to work on this next week.

https://twitter.com/rustdesk/status/1788905463678951787

**Comment by rustdesk** [comment]
https://github.com/rustdesk/rustdesk/pull/8066

**Comment by rustdesk** [comment]
https://rustdesk.com/docs/en/self-host/client-configuration/advanced-settings/
